src/sentry/notifications/utils/__init__.py

```diff
@@ -495,10 +493,13 @@ def send_activity_notification(notification: ActivityNotification | UserReportNo
         notify(provider, notification, participants, shared_context, extra_context)
 
 
-def get_replay_id(event: Event) -> str | None:
-    evidence_replay_id = event.occurrence.evidence_data.get("replay_id", "")
+def get_replay_id(event: Event, group: Group) -> str | None:
     tags_replay_id = event.get_tag("replay_id")
```

Comment:
the tag would be replayId, not `replay_id` I believe. I'm also thinking we need to verify that the replay exists, similar to how we do on the replay details page.

Reply:
let me see if this task is instrumented and how perf sensitive it is

Reply:
asked in slack. if its okay to do this, will be trivial to add.

Reply:
I was thinking a lot about verifying the replay id, and I actually went back yesterday and updated the link + copy to instead point to `/issues/123/replays` and say "See a replay of this issue".
I didn't pass the copy by jasmin yet, it's not the best.

But it sidesteps having to verify this specific replay, while still putting a link in here that is more likely to have what people are expecting.

Reply:
yeah, this makes sense. we can do a quick verification if its easy, and just ship this if its not. I just worry in the case of say a customer being quota rate limited, all of their replays won't exist, so it could lead to a confusing experience.

but this can also be fixed by updating copy / providing a guide to getting more quota etc., which we probably want to do anyways.

Reply:
Actually, another idea we could do is:

create a new url with the params: `<groupId>` and `<replayId>`, so maybe it's called `/issues/<gid>/replay-redirect/<rid>`

When the use visits this url, we look up the replayId, if it exists we 301 redirect to the replay details page.
if the replayid doesn't exist we 301 (maybe 302?) to `/issues/<gid>/replays/`. So the lookup happens on demand.